src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Equality.cs

```diff
@@ -403,6 +411,31 @@ public static Vector256<ushort> Load256(ref byte ptr)
                 (Vector128<ushort> lower, Vector128<ushort> upper) = Vector128.Widen(Vector128.LoadUnsafe(ref ptr));
                 return Vector256.Create(lower, upper);
```

Comment:
```suggestion
                return Vector256.WidenLower(Vector128.LoadUnsafe(ref ptr).ToVector256Unsafe());
```

This results in better codegen when `Avx2` is available.

Reply:
This suggests a set of missing `System.Runtime.Intrinsics.Vector256` APIs:
```csharp
public static System.Runtime.Intrinsics.Vector256<ushort> Widen (System.Runtime.Intrinsics.Vector128<byte> source);
```

Reply:
> This results in better codegen when `Avx2` is available.

Codegen on `arm64` is pretty bad though.